src/Maths/Silk.NET.Intrinsics/Avx/AvxRegister.Helpers.cs

```diff
@@ -0,0 +1,40 @@
+// This file is part of Silk.NET.
+// 
+// You may modify and distribute Silk.NET under the terms
+// of the MIT license. See the LICENSE file for details.
+
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.Arm;
+using System.Runtime.Intrinsics.X86;
+
+namespace Silk.NET.Intrinsics.Avx
+{
+    public partial struct AvxRegister
+    {
+        public static WorkUnitFlags Flags { get; } = GetFlags();
+        public static WorkUnitFlags Flags128F { get; } = Flags | WorkUnitFlags.Vector128 | WorkUnitFlags.TypeFloat;
```

Comment:
Framework design guidelines recommends using the full non language specific name so there is no ambiguity.

That is, these should be `Flags128Single`, `Flags128UInt64`, etc.

Reply:
Gotcha, will rename though these are meant to be private properties that for some reason I left public.